modules/cluster-and-ha-management/pages/set-up-crr.adoc

```diff
@@ -50,7 +50,7 @@ Run the following commands on the DR cluster to enable CRR on the DR cluster.
 
 [source.wrap,console]
 ----
-# Enable Kafka Mirrormaker
+# Enable cross-region replication
```

Comment:
There are also new commands to check and update the configs, do you think we should educate the user on those?

Reply:
Added a section to demonstrate the usage of "gadmin crr config" and "gadmin crr update" commands.
L101 ～L142